build-support/scripts/gen-changelog.sh

```diff
@@ -66,8 +71,10 @@ esac
 
 msg=""
 
-read -ep $'Please enter the changelog message:\n' msg
+read -erp $'Please enter the changelog message:\n' msg
 
 echo -e "\`\`\`release-note:$type\n$msg\n\`\`\`" >>"$filename"
 
-cat .changelog/$pr_number.txt
+echo -e "\nChangelog added to $filename. Contents:\n"
+
+cat ".changelog/$pr_number.txt"
```

Comment:
Nitpick, but this is broken for enterprise changelogs as their filenames start with "_".

Reply:
I think should probably be $filename instead?

Reply:
yup it should be!

Reply:
done and done